tox.ini

```diff
@@ -48,28 +48,15 @@ commands =
     ruff {[vars]all_path}
     black --check --diff {[vars]all_path}
 
-[testenv:static-{charm,unit,integration}]
+[testenv:static-charm]
 description = Run static analysis checks
 setenv =
     charm: MYPYPATH = {toxinidir}/lib
-    unit: MYPYPATH = {toxinidir}/lib:{[vars]src_path}
-    integration: MYPYPATH = {toxinidir}/lib:{[vars]src_path}:{[vars]tst_path}/integration
 deps =
-    mypy
-    types-PyYAML
-    types-setuptools
-    types-toml
-    charm,unit: {[testenv:unit]deps}
-    integration: {[testenv:integration]deps}
-commands =
-    charm: mypy {[vars]src_path} {posargs}
-    unit: mypy {[vars]tst_path}/unit {posargs}
-    integration: mypy {[vars]tst_path}/integration {posargs}
-
-[testenv:static-lib]
```

Comment:
The empty tox env was needed because the centralized CI fails if it's absent.
Is this no longer the case?

Reply:
It looks like it, yes; `tox` runs successfully for me when specifying a non-existing environment:
```
$ tox -e aaa
  aaa: OK (7.72 seconds)
  congratulations :) (7.78 seconds)
```